service_mesh/v2x/ossm-deploy-production.adoc

```diff
@@ -0,0 +1,14 @@
+[id="ossm-production"]
+include::modules/ossm-document-attributes.adoc[]
+= Deploying to production 
+:context: ossm-architecture
+toc::[]
+
+When you're ready to deploy your service mesh to production, you must configure your control plane, tracing, and security certificates to meet production requirements. 
```

Comment:
I'm not familiar enough with service mesh to make this call. But I know a lot of products avoid contractions. If service mesh is cool with it, keep `you're.`